poll.js

```diff
@@ -0,0 +1,150 @@
+'use strict'
+var imgs = ['bag.jpg',
+            'banana.jpg',
+            'bathroom.jpg',
+            'boots.jpg',
+            'breakfast.jpg',
+            'bubblegum.jpg',
+            'chair.jpg',
+            'cthulhu.jpg',
+            'dog-duck.jpg',
+            'dragon.jpg',
+            'pen.jpg',
+            'pet-sweep.jpg',
+            'scissors.jpg',
+            'shark.jpg',
+            'sweep.png',
+            'tauntaun.jpg',
+            'unicorn.jpg',
+            'usb.gif',
+            'water-can.jpg',
+            'wine-glass.jpg'];
+
+var choices = [];
+choices.push(document.getElementById('picture0'));
+choices.push(document.getElementById('picture1'));
+choices.push(document.getElementById('picture2'));
+var results = document.getElementById('results');
+var rlist = document.getElementById('resultslist');
+
+/**
```

Comment:
If you're going to add jsdoc comments to describe your functions, take the time to fill them out, otherwise they're meaningless.